events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js

```diff
@@ -0,0 +1,747 @@
+//  triviaMasterClientStandard.js
+//
+//  Created by Rebecca Stankus on 06/11/18
+//  Copyright 2018 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+//  This is the standard version of "Last Avatar Standing" Trivia 
+//  that draws from a third-party database of thousands of questions.
+
+/* global EventBridge Users AccountServices */
+
+(function() {
```

Comment:
coding standards: missing comments throughout the file

Reply:
since we're not rearchitecting everything right now, don't bother.

Reply:
I will adhere to this going forward.